src/compiler/checker.ts

```diff
@@ -40095,10 +40095,10 @@ namespace ts {
                         }
                         // ECMA262: 15.2.1.1 It is a Syntax Error if the ExportedNames of ModuleItemList contains any duplicate entries.
                         // (TS Exceptions: namespaces, function overloads, enums, and interfaces)
-                        if (flags & (SymbolFlags.Namespace | SymbolFlags.Interface | SymbolFlags.Enum)) {

```

Comment:
does this mean `export interface I { }; export interface I { }` is no longer allowed?

Reply:
i just checked and that still works, i assume there's already a test for this somewhere but i can't find it. if not, i can add one if you'd like

Reply:
Yes, please. I'm not confident in our existing test coverage.

Reply:
done